backend/src/main/java/ro/utcn/sd/a3/controller/TeacherController.java

```diff
@@ -0,0 +1,18 @@
+package ro.utcn.sd.a3.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+import ro.utcn.sd.a3.entity.Teacher;
+import ro.utcn.sd.a3.service.TeacherUserDetailsService;
+
+@RestController
+@RequiredArgsConstructor
+public class TeacherController {
+    private final TeacherUserDetailsService service;
```

Comment:
This seems to be a left-over class from the example... It means that you also have the entities, services, etc in your project (Teacher, TeacherService, etc).